src/lightning_app/runners/cloud.py

```diff
@@ -106,6 +106,90 @@ def _to_clean_dict(swagger_object, map_attributes):
 class CloudRuntime(Runtime):
     backend: Union[str, CloudBackend] = "cloud"
 
+    def open(self, name: str, cluster_id: Optional[str] = None):
+        """Method to open a CloudSpace with the root folder uploaded."""
+        try:
+            # Check for feature support
+            user = self.backend.client.auth_service_get_user()
+            if not user.features.code_tab:
+                print(
+                    "The `lightning open` command has not been enabled for your account. "
+                    "To request access, please contact support@lightning.ai"
+                )
+                sys.exit(1)
```

Comment:
can you use rich.print and have this show in red text or something?

Reply:
Done :smiley: